src/Numerics/ODESolvers/AdditiveRungeKuttaMethod.jl

```diff
@@ -522,6 +524,64 @@ end
     end
 end
 
+"""
+    ARK1ForwardBackwardEuler(f, l, backward_euler_solver, Q; dt, t0,
+                             split_explicit_implicit, variant)
+
+This function returns an [`AdditiveRungeKutta`](@ref) time stepping object,
+see the documentation of [`AdditiveRungeKutta`](@ref) for arguments definitions.
+This time stepping object is intended to be passed to the `solve!` command.
+
+This uses a first-order-accurate two-stage additive Runge--Kutta scheme
+by combining a forward Euler explicit step with a backward Euler implicit
+correction.
+"""
+function ARK1ForwardBackwardEuler(
+    F,
+    L,
+    backward_euler_solver,
+    Q::AT;
+    dt = nothing,
+    t0 = 0,
+    split_explicit_implicit = false,
+    variant = LowStorageVariant(),
+) where {AT <: AbstractArray}
+
+    @assert dt !== nothing
+
+    T = eltype(Q)
+    RT = real(T)
+
+    RKA_explicit = [
+        RT(0) RT(0)
+        RT(1) RT(0)
+    ]
+    RKA_implicit = [
+        RT(0) RT(0)
+        RT(0) RT(1)
+    ]
+
+    RKB = [RT(0), RT(1)]
+    RKC = [RT(1), RT(1)]
```

Comment:
Just checking, did you mean to evaluate the rhs for the forward Euler step at `t_{n+1}` ?

Reply:
This was a mistake. `RKC` should be `[0, 1]`. I fixed this